lib/biran/configurinator.rb

```diff
@@ -30,12 +30,12 @@ def initialize
     end
 
     def file_tasks
-      files_to_generate.keys
+      config_files_to_generate.keys
```

Comment:
We spend a lot of time on the naming of this part... I don't understand the name change. This is going to be used for not only config files, but other files outside of config and maybe not used for config at all? The vhost file is not really a config file, however, it could be argued either way.

Reply:
this is a config Gem, every file to be generated is a config file... vhost is not config? it has `conf` in the extension... so not sure if we making this very narrow distinctions help us

Reply:
As stated, we went round and round in the original PR over naming of this. There is no need to add "config" to the beginning of a method in a config object. Its redundant. My argument is that the gem is used to help configure your app. Maybe that is creating config files, maybe that is generating files needed for the app to work properly that aren't really configuring anything. Its a context thing. I just don't like the naming and I still think its an unnecessary change since all it appears to be doing is wrapping a default value that is more easily removed as a default value.

Reply:
This is a delegated method, so it is a good pattern, it is the files to generate from the. Config object